src/wallet/wallet.cpp

```diff
@@ -3311,6 +3311,22 @@ size_t CWallet::KeypoolCountExternalKeys()
     return setExternalKeyPool.size() + set_pre_split_keypool.size();
 }
 
+bool CWallet::NoPrivkeysAndKeypoolEmpty(bool internal)
+{
+    if (!IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS)) {
+        return false;
+    }
+
+    {
+        LOCK(cs_wallet);
+        if (internal) {
+            return setInternalKeyPool.size() == 0;
```

Comment:
mu-nit: add commit for KeypoolCountInternalKeys

Reply:
I think that's unnecssary.